### PR TITLE
[ADD] (Optional) permission boundary for task execution IAM role

### PR DESCRIPTION
**Pull request by ovcharenko**
This is related to https://github.com/cn-terraform/terraform-aws-ecs-fargate-task-definition/pull/8

~~Need to update `td` version dependency.~~ Done

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -13,9 +13,9 @@ module "ecs-cluster" {
 # ECS Task Definition
 #------------------------------------------------------------------------------
 module "td" {
-  source  = "cn-terraform/ecs-fargate-task-definition/aws"
-  version = "1.0.18"
-  # source  = "../terraform-aws-ecs-fargate-task-definition"
+  //  source  = "cn-terraform/ecs-fargate-task-definition/aws"
```
Comment (jnonino) [inline main.tf:16]:
```suggestion
  source  = "cn-terraform/ecs-fargate-task-definition/aws"
```

File: main.tf [hunk 2]
```diff
@@ -13,9 +13,9 @@ module "ecs-cluster" {
...
+  //  version = "1.0.18"
```
Comment (jnonino) [inline main.tf:17]:
```suggestion
  version = "1.0.19"
```

File: main.tf [hunk 3]
```diff
@@ -13,9 +13,9 @@ module "ecs-cluster" {
...
+  source = "../terraform-aws-ecs-fargate-task-definition"
```
Comment (jnonino) [inline main.tf:18]:
```suggestion
  // source = "../terraform-aws-ecs-fargate-task-definition"
```

**Review by jnonino — CHANGES_REQUESTED** [review]

File: main.tf (hunk 3 above)
Comment (ovcharenko) [reply main.tf]: Updated in 1bf70c7

File: main.tf (hunk 2 above)
Comment (ovcharenko) [reply main.tf]: Updated in 1bf70c7

File: main.tf (hunk 1 above)
Comment (ovcharenko) [reply main.tf]: Updated in 1bf70c7

**Review by jnonino — APPROVED** [review]

**Pull request merged**